node/src/BaseClient.ts

```diff
@@ -3694,7 +3694,7 @@ export class BaseClient {
     public close(errorMessage?: string): void {
         this.isClosed = true;
         this.promiseCallbackFunctions.forEach(([, reject]) => {
-            reject(new ClosingError(errorMessage));
+            reject(new ClosingError(errorMessage || ""));
```

Comment:
It should be "cancelled" or or something like this

Reply:
wdym? I just fixed it in case theres no errMessage